### PR TITLE
feat: adding tutor-contrib-pod-autoscaling plugin to contrib index

### PR DESCRIPTION
**Pull request by jfavellar90**
The tutor plugin link: https://github.com/eduNEXT/tutor-contrib-pod-autoscaling

### CODE REVIEW

File: contrib/nightly/plugins.yml [hunk 1]
```diff
@@ -6,3 +6,11 @@
   description: |
     Tutor plugin to enable execution of untrusted code in secure sandboxes.
     In particular, this plugin is required to run Python-evaluated input XBlocks.
+- name: pod-autoscaling
+  src: -e git+https://github.com/eduNEXT/tutor-contrib-pod-autoscaling@v15.0.0#egg=tutor-contrib-pod-autoscaling
```
Comment (regisb) [inline contrib/nightly/plugins.yml:10]: Are you sure you want to install from v15.0.0 tag, and not the nightly branch?

File: contrib/nightly/plugins.yml [hunk 2]
```diff
@@ -6,3 +6,11 @@
   description: |
     Tutor plugin to enable execution of untrusted code in secure sandboxes.
     In particular, this plugin is required to run Python-evaluated input XBlocks.
+- name: pod-autoscaling
```
Comment (regisb) [inline contrib/nightly/plugins.yml:9]: Add an empty line just above (and in the olive file as well).

**Review by regisb — CHANGES_REQUESTED** [review]
Thanks for your contribution!

File: contrib/nightly/plugins.yml (hunk 2 above)
Comment (jfavellar90) [reply contrib/nightly/plugins.yml:10]: @regisb addressed

File: contrib/nightly/plugins.yml (hunk 1 above)
Comment (jfavellar90) [reply contrib/nightly/plugins.yml:11]: @regisb I just have a question here, `nightly` will always follow `main`? Is it like the latest version? According to [this](https://docs.tutor.overhang.io/tutorials/nightly.html), that's the case, I just wanted to make sure I'm not misunderstanding the purpose of the nightly branch

Comment (jfavellar90) [reply contrib/nightly/plugins.yml]: @regisb fixed

**Comment by jfavellar90** [comment]
@regisb do you think it is ready to merge?

**Review by regisb — APPROVED** [review]
LGTM!

**Comment by regisb** [comment]
Users should now be able to install the plugin with `tutor plugins install pod-autoscaling`.

**Pull request merged**